var/spack/repos/builtin/packages/rocm-opencl/package.py

```diff
@@ -10,15 +10,25 @@ class RocmOpencl(CMakePackage):
     """OpenCL: Open Computing Language on ROCclr"""
 
     homepage = "https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime"
-    url      = "https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/archive/roc-3.5.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    def url_for_version(self, version):
+        if version == Version('3.5.0'):
+            return "https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/archive/roc-3.5.0.tar.gz"
```

Comment:
Any chance the AMD devs can rename this tarball to make the URL consistent across versions?

Reply:
i've already created an issue elsewhere: https://github.com/ROCm-Developer-Tools/ROCclr/issues/18, but no response so far. let me do the same for this package

Reply:
https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/issues/126

Reply:
@adamjstewart @haampie all the future  ROCm source package will be prefixed with _rocm_ instead of _roc_. This change is made to keep archive name consistency across all the ROCm components.